public/js/checkin.js

```diff
@@ -14,8 +14,29 @@ window.HLRDESK.init.checkin = function() {
     return $(node).data("date")||$(node).text()
   }
   $(".checkIN").tablesorter( {textExtraction: myTextExtraction} );
-  
+  function checkInItems(evt){
+    evt.preventDefault();
+    var selected = document.querySelectorAll('.selected');
+    for(var i = 0; i<selected.length; i++){
+      selected[i].parentNode.removeChild(selected[i]);
```

Comment:
This may be a misinterpretation, but isn't the parents child that has the same name the same thing? How about ".remove()"? Or is this not what that does?